cleanlab_studio/studio/enrichment.py

```diff
@@ -92,3 +107,202 @@ def to_dict(self) -> Dict[str, Any]:
             "created_at": self.created_at,
             "updated_at": self.updated_at,
         }
+
+    def preview(
+        self,
+        options: EnrichmentOptions,
+        *,
+        new_column_name: str,
+        indices: Optional[List[int]] = None,
+        disable_warnings: bool = False,
```

Comment:
This isn't in the PRD. IIRC the client-side enrichment had this in order to deal with how every single row that was being enriched could raise exceptions if it and only it had issues. This is not a problem for us since row-level issues are added added to the `log` field rather than causing a client-side failure.
```suggestion
```

Reply:
done.